### ISSUE TITLE
[Adaptive Cards - "InputswithValidation.JSON"]: Radio buttons and checkboxes on the card are Tab-key navigable.

### ISSUE THREAD

**Issue by HasnuddinMohammed** [microsoft/AdaptiveCards#8740]
### Target Platforms

UWP

### SDK Version

1.2.2310.16007

### Application Name

Adaptive Card Visualizer

### Problem Description

**Test Environment:**
Platform: Win32
OS version: 23H2 (Build 25967.1010)
Application: Adaptive Card Visualizer
Application Version: 1.2.2310.16007
Screen reader: Narrator

**Repro steps:​**

1. Open the Adaptive Card Visualizer application.
2. Navigate the Activity Update. JSON card.
3. Navigate and activate the set due date and observe the issue

**Actual Result:**
Checkboxes buttons under "What colors do you want? (Multiselect)" are tab key navigable. 

**Expected result:**
"What colors do you want? (Multiselect)" checkboxes should be arrow key-navigable instead of tab key navigable. 

**User Experience:**
For better user's experience, "What colors do you want? (Multiselect) need to navigable with arrow keys. 

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/79564581/8052bae8-1833-4b98-a8e5-4302896ac679

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Since the product is getting deprecated, hence closing this issue.
